### ISSUE TITLE
Add a way for users to define x y columns in the data if not automatically identified

### ISSUE THREAD

**Issue by tbock** [USEPA/Scribe-Data-View-Tool#15]
For example: We load data and we find no spatial information in the normal locations
A dialog or flag shows up saying No Geospatial Points click here to identify spatial columns in the data if available.  Once the users selects the columns we attempt reload the data on to the map using this new information.

Cheryl would like to allow the user to leave a comment when adding a point, describing info about the point. Thus, a dialog is the likely approach for this issue.

**Comment by tommckennon** [comment]
This can be done in Ag Grid by enabling table cell editing. 
Reference: https://www.ag-grid.com/javascript-grid-cell-editing/

**Comment by tommckennon** [comment]
Postponed until Cheryl brings up updating Scribe data again, and then we'll discuss an idea on if and how it can be done.

CC @jlherring

**Comment by tbock** [comment]
Clarification.  Just provide a way for users to select which columns represent x and y if no geospatial data is found.

**Comment by tommckennon** [comment]
Can I do something like this for this enhancement:
![image](https://user-images.githubusercontent.com/28331071/97373717-d348c780-1873-11eb-81a1-c8529671e3e2.png)

Essentially change the Visible Columns dialog to include more column/field property toggles? Have the additional geospatial options be set only once?

CC @jlherring

**Comment by jlherring** [comment]
narrowed scope to purely Scribe data; won't be relevant unless that changes again
